torch_glow/src/binding.cpp

```diff
@@ -23,6 +23,12 @@
 #include "PyTorchCommon.h"
 #include "PyTorchModelLoader.h"
 
+#include "TorchGlowTraining.h"
+#include <pybind11/pybind11.h>
+/// Required include files for a proper binding TorchGlowTrainingWrapper class.
+#include <pybind11/stl.h>
+#include <torch/csrc/utils/pybind.h>
```

Comment:
why is this needed?

Reply:
As comment says, without those two files, pybind doesn't recognize correspondently std::vector and at::Tensor types.